src/Coord.py

```diff
@@ -1,18 +1,19 @@
-import chess_enums
+from chess_enums import Space
 
-
-class Action:
+class Coord:
     c: int
     r: int
 
-    def __init__(self, par_space: chess_enums.Space) -> None:
-        self.c = par_space / 8
-        self.r = par_space % 8
-
     def __init__(self, par_c: int, par_r: int) -> None:
         self.c = par_c
         self.r = par_r
 
     @classmethod
-    def __call__(self) -> chess_enums.Space:
-        return chess_enums.Space(8*self.c + self.r)
+    def fromSpace(cls, par_space: Space) -> None:
+        c = par_space / 8
```

Comment:
This gives a type error when I try it. Have you verified that all your new code compiles in Python? The way I've been testing that is that once I've written a function or class, I import the file in the IDE's interactive window, and try printing the class or output of the function. If there's a compile error, it should show up when I print that. For simple functions you can also test the validity of the output. For example, after importing the necessary modules, you can check that "print(Coord.Coord.fromSpace(Space.B2).c, Coord.Coord.fromSpace(Space.B2).r)" outputs "1 1"

Reply:
Looks like I missed some compilation errors due to misuse of the @classmethod tag, that's been fixed now

Reply:
What does IntEnum have that Enum doesn't have? (regular Enum still has the value member)

Reply:
Ah nevermind, just looked it up. Seems we don't need .value with IntEnums.